libraries/common/cs/devtools-utils.js

```diff
@@ -41,6 +41,13 @@ export const getTopBlocks = (workspace) => {
   return topBlocks;
 };
 
+export const autoPositionComment = (comment) => {
+    comment.needsAutoPositioning_ = true;
+    comment.autoPosition_()
```

Comment:
This should check if `autoPosition_()` exists - only comments attached to blocks have it. Currently, an error is thrown if there are any standalone comments in the workspace:
![image](https://github.com/user-attachments/assets/3ed59d5e-4398-4e9a-88c7-d387386d4f5f)

Reply:
Thank you for catching this!